lib/store/peer.ex

```diff
@@ -50,4 +50,37 @@ defmodule Elixium.Store.Peer do
       end
     end
   end
+
+  @spec find_potential_peers :: List | :not_found
+  def find_potential_peers do
+    case load_known_peers() do
```

Comment:
This is upto you, but if were mathcing off an empty list with a case, it may be tidier to use ```with-else```, thoughts?

Reply:
With is usually used for chaining values together and ensuring they conform to set rules -- we're not doing that here so I feel it'd look more confusing than just using a case statement.